apps/mdn/mdn-aws/k8s/Makefile

```diff
@@ -178,6 +178,25 @@ export ELB_S3_LOGGING_BUCKET ?= mdn-elb-logs
 # ELB_S3_LOGGING_PREFIX can't end with a slash!
 export ELB_S3_LOGGING_PREFIX ?= ${TARGET_ENVIRONMENT}
 
+### New Relic configuration
+export NEW_RELIC_SECRETS_NAME ?= newrelic-secrets
+export NEW_RELIC_LICENSE_KEY ?= foo
+export NEW_RELIC_LICENSE_KEY_BASE64 ?= $(shell echo -n "${NEW_RELIC_LICENSE_KEY}" | base64)
+export NEW_RELIC_CONFIG_FILE ?= /app/etc/newrelic.ini
+export NEW_RELIC_MONITOR_MODE ?= 1
+export NEW_RELIC_LOG ?= stderr
+export NEW_RELIC_BROWSER_MONITOR_ENABLE ?= false
+
+# New Relic app names
+export NEW_RELIC_MONITOR_SUFFIX ?= ${TARGET_ENVIRONMENT}-${K8S_CLUSTER_SHORT_NAME}
+export NEW_RELIC_ROLLUP_NAME ?= kuma-${NEW_RELIC_MONITOR_SUFFIX}
+export NEW_RELIC_WEB_NAME ?= "kuma-web-${NEW_RELIC_MONITOR_SUFFIX};${NEW_RELIC_ROLLUP_NAME}"
+export NEW_RELIC_API_NAME ?= "kuma-api-${NEW_RELIC_MONITOR_SUFFIX};${NEW_RELIC_ROLLUP_NAME}"
+export NEW_RELIC_KUMASCRIPT_NAME ?= "kuma-kumascript-${NEW_RELIC_MONITOR_SUFFIX};${NEW_RELIC_ROLLUP_NAME}"
```

Comment:
The node.js is comma-separated, so the semi-colon ``;`` needs to be a comma ``,``.